common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -2579,6 +2580,15 @@ object CelebornConf extends Logging {
       .booleanConf
       .createWithDefault(false)
 
+  val WORKER_PUSH_COMPOSITEBUFFER_MAXCOMPONENTS: ConfigEntry[Int] =
+    buildConf("celeborn.worker.push.compositeBuffer.maxComponents")
+      .internal
+      .categories("worker")
+      .version("0.3.0")
+      .doc("Max components in CompositeByteBuf for FileWriter's flushBuffer.")
```

Comment:
better to explain a bit more about this configuration

Reply:
done